sale_probability_amount/views/sale_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_quotation_tree" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_quotation_tree" />
+        <field name="arch" type="xml">
+            <field name="amount_total" position="after">
+                <field name="expected_amount_cur" sum="Total" widget="monetary" />
+                <field name="percentage" optional="hide" />
+            </field>
+        </field>
+    </record>
+
+    <record id="view_order_form" model="ir.ui.view">
+            <field name="model">sale.order</field>
+            <field name="inherit_id" ref="sale.view_order_form" />
+            <field name="arch" type="xml">
+                <field name="order_line" position="after">
+                    <group
+                    colspan="4"
+                    attrs="{'invisible':[('state', 'not in', ['draft', 'sent'])]}"
+                >
```

Comment:
```suggestion
                    <group
                        colspan="4"
                        attrs="{'invisible':[('state', 'not in', ['draft', 'sent'])]}"
                    >
```

Reply:
The pre-commit prettier hook seems to force this indentation.